libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.spec.ts

```diff
@@ -354,23 +376,56 @@ describe('Summary Action Bar component', () => {
         fixture.detectChanges();
         mockMediaQueryService.fire(SkyMediaBreakpoints.xs);
         fixture.detectChanges();
-        debugElement
-          .query(By.css('.sky-summary-action-bar-details-collapse button'))
-          .nativeElement.click();
+        clickCollapseButton(debugElement);
         fixture.detectChanges();
         tick();
         fixture.detectChanges();
         expect(cmp.summaryActionBar.isSummaryCollapsed).toBeTruthy();
         expect(cmp.summaryActionBar.slideDirection).toBe('up');
-        debugElement
-          .query(By.css('.sky-summary-action-bar-details-expand button'))
-          .nativeElement.click();
+        clickExpandButton(debugElement);
         fixture.detectChanges();
         tick();
         fixture.detectChanges();
         expect(cmp.summaryActionBar.isSummaryCollapsed).toBeFalsy();
         expect(cmp.summaryActionBar.slideDirection).toBe('down');
       }));
+
+      it(`should move focus to the collapsed summary's chevron after collapsing`, async () => {
```

Comment:
@Blackbaud-SteveBrush This and the next test are the new ones. Everything else here was general cleanup that I did as this file was hard to read in a lot of places and had various `any` and deprecation issues. I'm happy to move the refactors to a different PR if you'd like.

Reply:
Love it.